density/density.py

```diff
@@ -508,12 +508,13 @@ def upload():
 
     density_data = json_data['data']
 
-    db_failed = db.insert_density_data(g.cursor, density_data)
-    if db_failed == 0:
+    db_success = db.insert_density_data(g.cursor, density_data)
+    if db_success:
         return 'Data successfully uploaded.', 200
     else:
-        return '%d rows were not inserted. Please try again or contact someone in ADI.' % db_failed, 500
+        return 'At least one of the records was not inserted. \
+            Please contact someone in ADI for more details.', 500
```

Comment:
So I personally think that `db.insert_density_data` should just throw an error, and we should have a generic Flask error handler to tell people to contact ADI.